app/controllers/admin/offices_controller.rb

```diff
@@ -2,5 +2,24 @@
 
 module Admin
   class OfficesController < Admin::ApplicationController
+    def create
```

Comment:
This seems to work well!

I think it might be better to take a slightly different approach.

If you were to instead:
- Make the `skip_api_validation` `attr_accessor` in `Address` public instead of private so that the virtual attribute could be set directly from attributes coming in from the `Address` section of the `Office` form
- Add `skip_api_validation` to the `AddressDashboard::ATTRIBUTES_TYPES` and `AddressDashboard::FORM_ATTRIBUTES` constants so the field appears in the `Address` form
- Add `skip_api_validation` to the list of permitted address attributes in `OfficePolicy` so the parameter is whitelisted
- Remove the `skip_api_validation?` alias from `Address` and define a method to handle converting the incoming boolean from something like `'1'` to `true` in the `Address` model like:
```
def skip_api_validation?
  ActiveModel::Type::Boolean.new.cast(skip_api_validation)
end
```

It would:
- Eliminate the need to override `OfficesController#create`
- Eliminate the `skip_confirmation <-> skip_api_validation` coupling between the `Address` and `Office` models (would not need the methods you added to `Office`)
- Not require using `send` to get the value of the private attribute
- Nest the option to `Skip api validation` inside of the `Address` section of the `Office` form (think this is preferable since the address validation is the one being skipped)

Bonus points if you want to figure out how to use a custom label in lieu of `Skip api validation` (maybe something like `Skip address geocoding`)


If you were to do the above, I think you could:
- Remove your `Admin::OfficesController` override
- Remove the added methods from `Office`
- Undo the changes to `OfficeDashboard`

You would need to update the specs you added to pass `skip_api_validation` as an Address attribute instead of passing `skip_confirmation` as an `Office` attribute.